docs/detections/prebuilt-rules-update-modified-unmodified.asciidoc

```diff
@@ -0,0 +1,116 @@
+[[prebuilt-rules-update-modified-unmodified]]
+== Update modified and unmodified Elastic prebuilt rules
+
+.Requirements
+[sidebar]
+--
+You must have an https://www.elastic.co/subscriptions/cloud[Enterprise] subscription to access this feature. If you have a Platinum subscription or lower, follow the guidelines in <<update-prebuilt-rules>> instead.
+--
+
+This page provides instructions for updating modified and unmodified prebuilt rules. You can also find information about <<rule-field-update-statuses,statuses>> or <<resolve-reduce-rule-conflicts,conflicts>> that you might encounter when updating rules. 
+
+To update rules:
+
+. Find *Detection rules (SIEM)* in the navigation menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+. In the *Rules* table, select the *Rule Updates* tab.
++
+NOTE: The *Rule Updates* tab doesn't appear if all your installed prebuilt rules are up to date.
++
+[role="screenshot"]
+image::images/prebuilt-rules-update.png[The Rule Updates tab on the Rules page]
+
+. (Optional) To examine the details of a rule's latest version before you update it, select the rule name. This opens the rule update flyout, where you can: 
+
+** Preview incoming updates: Select the *Elastic update overview* tab to view rule changes field by field, or the *JSON view* tab to view changes for the entire rule in JSON format. 
++
+Both tabs display side-by-side comparisons of the *Current rule* (what you currently have installed) and the *Elastic update* version (what you can choose to install). Deleted characters are highlighted in red; added characters are highlighted in green.
++
+** Compare different versions of a rule field: Use the **Diff view** drop-down menu to compare different versions of a rule field. For example, compare the changes that you made to the current version of the field with changes that will be applied from the incoming Elastic update.
++
+NOTE: If you haven't updated the rule in a while, its original version might be unavailable for comparison. Instead, you will only have access to the rule's current version and the incoming Elastic update. You can avoid this by updating prebuilt rules more frequently. 
+
+** Check the update status: View the status of the entire rule update and for <<rule-field-update-statuses,each field that's being changed>>. 
+
+** Address update conflicts: Find and address conflicts that <<resolve-reduce-rule-conflicts, need additional attention>>. 
+
+** Edit the final update: Change the update that will be applied to the field when you update the rule. To change the update, go to the *Final update* section, make your changes, and then save them.
++
+IMPORTANT: Elastic updates containing a rule type change cannot be edited. Before updating the rule, duplicate it if you need to record changes that you made to other rule fields. 
```

Comment:
suggestion: "duplicate it if you need to record changes that you made to other rule fields. " -> duplicate it if you need to record changes that you made to the rule fields.

Reply:
```suggestion
IMPORTANT: Elastic updates containing a rule type change cannot be edited. Before updating the rule, duplicate it if you need to record changes that you made to the rule fields. 
```